templates/blog-page.html

```diff
@@ -1,7 +1,16 @@
 {% extends "base.html" %}
-    {% block title %}
-		<title>Blog | {{blog_post.title}}</title>
-    {% endblock %}
+{% block title %}
+    <title>Blog | {{blog_post.title}}</title>
+{% endblock %}
+
+{% block og_params %}
+    <!-- OG params for sharable content -->
+    <meta property="og:title" content="Blog | {{ blog_post.title }}" />    
+    <meta property="og:type" content="website" />
+    <meta property="og:url" content="https://whotracks.me/blog/{{blog_post.filename}}" />
```

Comment:
Could the `url` come from the python code generating the site? It probably does not change much, but since we already know the structure in the post, that looks like duplicated logic.